### PR TITLE
Add front-end decision log

### PR DESCRIPTION
**Pull request by dzhavat**
## References

#-

### CODE REVIEW

File: docs/front-end/decision-log.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### Option A - Use the same repository (monorepo) to host all front-ends (FEs), backend-for-frontends (BFFs) and the Design System
+
+Pros
+
+- A single code repository encourages team collaboration
+- Easier code reuse across projects
+- Single CI/CD pipeline
+- Better code visibility across teams
+- Mature tools to support the work
+- Single version of Angular across FEs and Design System
```
Comment (frontend-specialisten) [inline docs/front-end/decision-log.md:20]:
```suggestion
- Single version of Angular and other third party libraries across FEs and Design System
```

File: docs/front-end/decision-log.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Decision log
+
+## 1. Hosting of front-ends (FEs), backend-for-frontends (BFFs) and Design System in Green Force ART
+
+Date: 2021-09-20
+
+Decision: **Option A**
```
Comment (frontend-specialisten) [inline docs/front-end/decision-log.md:7]:
```suggestion
Decision: **Option A** has been chosen as **option B** can be applied later on, if necessary  
```

**Review by frontend-specialisten — CHANGES_REQUESTED** [review]

**Review by frontend-specialisten — APPROVED** [review]

File: docs/front-end/decision-log.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Decision log
+
+## 1. Hosting of front-ends (FEs), backend-for-frontends (BFFs) and Design System in Green Force ART
+
+Date: 2021-09-20
+
+Decision: **Option A** has been chosen as **option B** can be applied later on, if necessary  
+
+Responsible: SSG, MEA, XALEH, XDZUS
```
Comment (Sondergaard) [inline docs/front-end/decision-log.md:9]: Please use our GitHub handles:
``` suggestion
Responsible: @Sondergaard, @MartinFHansen, @frontend-specialisten, @dzhavat
```

**Review by Sondergaard — CHANGES_REQUESTED** [review]

File: docs/front-end/decision-log.md (hunk 3 above)
Comment (dzhavat) [reply docs/front-end/decision-log.md]: Thanks for the clarification. Makes sense when we're on GitHub.

**Comment by dzhavat** [comment]
@Sondergaard Do you have further comments to this PR?

**Review by Sondergaard — APPROVED** [review]

**Comment by Sondergaard** [comment]
> @Sondergaard Do you have further comments to this PR?

I don't have any further comments on this PR

**Pull request merged**